iroha_client/tests/cucumber.rs

```diff
@@ -1,874 +0,0 @@
-#![allow(missing_docs, clippy::restriction, clippy::too_many_lines)]
-
-use std::{convert::Infallible, panic::UnwindSafe, thread, time::Duration};
-
-use async_std::task::{self, JoinHandle};
-use async_trait::async_trait;
-use cucumber_rust::{Cucumber, World as CucumberWorld};
-use futures::executor;
```

Comment:
Should we remove this test?

Reply:
Yes, we planned to remove cucumber as far as I remember as it is not used anymore. I just did it not to fix all the internal code there.